shared/src/logging.rs

```diff
@@ -126,8 +126,8 @@ impl TrxContextBuilder {
             response_headers: None,
             response_code: None,
             status_group: None,
-            cage_name: None,
-            cage_uuid: None,
+            enclave_name: None,
+            enclave_uuid: None,
```

Comment:
While we're changing it would it make sense to change this to just `name` and `uuid` or `resource_name` and `resource_uuid` as the type is already defined by the `trx_type`?

Reply:
yep fair shout!